README.md

```diff
@@ -1,33 +1,3 @@
-
-Eclipse Platform also provides a language-independent debug infrastructure that enables developers to debug programs written in different languages. Additionally, the platform includes infrastructure for distributed multi-user versioned resource management, which allows for collaborative development and version control of resources.
-
-Overall, Eclipse Platform is an essential set of tools and services for developers who need a robust and flexible platform for building complex software applications. Its comprehensive set of frameworks and common services ensure that developers can create high-quality software that meets the needs of their users. As the base framework for the Eclipse IDE and many other rich client applications, Eclipse Platform has a proven track record of providing a stable and reliable development platform that supports the creation of powerful software applications.
-
-![workbench](https://raw.githubusercontent.com/eclipse-platform/eclipse.platform.common/master/bundles/org.eclipse.platform.doc.isv/guide/images/workbench.png)
-
-See also https://projects.eclipse.org/projects/eclipse.platform and https://eclipse.org/eclipse
-
-## Reporting issues
-
-The Eclipse Platform project is split into multiple Git repositories, which are all part of this organization. If you face an issue and have a sense of which particular GitHub repository is most related, you can open your issue against that repository. If you're unsure, you can open an issue against this current repositories and the issue will then be moved as best by maintainers.
-
-e.g.
-- [New SWT issue](https://github.com/eclipse-platform/eclipse.platform.swt/issues/new)
-- [New Platform issue](https://github.com/eclipse-platform/eclipse.platform.ui/issues/new)
-
-## Contributing
-
-[Contributions are always welcome!](https://github.com/eclipse-platform/.github/blob/main/CONTRIBUTING.md)
-
-Please bear in mind that this project is almost entirely developed by volunteers. If you do not provide the implementation yourself (or pay someone to do it for you), the bug might never get fixed. If it is a serious bug, other people than you might care enough to provide a fix.
-
-## Code of Conduct
-🤝 This project is governed by the Eclipse Foundation [Code of Conduct](https://github.com/eclipse-platform/.github/blob/main/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report any unacceptable behavior to [conduct@eclipse-foundation.org](mailto:conduct@eclipse-foundation.org).
-
+See https://github.com/eclipse-platform/.github/blob/main/profile/README.md
```

Comment:
Instead of the technical source file, we could point to the user-friendly URL https://github.com/eclipse-platform.

Reply:
True. I have updated the URL.